networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +244,47 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+@not_implemented_for('directed','multigraph')
+def chordal_simplicial_vertex(G):
+    """Returns the simplicial vertex of the chordal graph G. 
+    
```

Comment:
Define a "simplicial vertex" here:

    A *simplicial vertex* of a chordal graph ``G`` is a node ``v`` whose closed neighborhood is a clique in ``G``.